content/en/sensitive_data_scanner/setup/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Setup
+disable_toc: false
+further_reading:
+- link: "/sensitive_data_scanner/setup/telemetry_data/"
```

Comment:
Same note as above - the setup pages are linked pretty prominently from the (short!) page copy. I guess I'm a little concerned that repeating the links is adding some unnecessary noise to the page?

Reply:
Thanks! I was wondering that myself. I'll remove.